go/libraries/doltcore/sqle/cluster/controller.go

```diff
@@ -97,6 +97,8 @@ type procedurestore interface {
 const (
 	DoltClusterRoleVariable      = "dolt_cluster_role"
 	DoltClusterRoleEpochVariable = "dolt_cluster_role_epoch"
+	// Since we fetch the keys from the other replicas we’re going to use a fixed string here.
```

Comment:
Tried to regurgitate when you said here but it might not be right lol

Reply:
Think this is fine...